frontend/src/admin/pages/[groupId]/contest/[id]/CreateNoticeModal.vue

```diff
@@ -0,0 +1,62 @@
+<script setup lang="ts">
+import Button from '@/common/components/Atom/Button.vue'
+import InputItem from '@/common/components/Atom/InputItem.vue'
+import Modal from '@/common/components/Molecule/Modal.vue'
+import TextEditor from '@/common/components/Organism/TextEditor.vue'
+import { ref } from 'vue'
+
+const props = defineProps<{
+  toggle: boolean
+  setToggle: (a: boolean) => void
+}>()
+const title = ref('')
+const problemId = ref('')
+</script>
+
+<template>
+  <Modal
+    :model-value="props.toggle"
```

Comment:
template 안에서는 `props` 참조 안해도 변수 불러올 수 있어요!  
위에 script에서 `const props = ` 지워도 동작 됩니다 🙂

```suggestion
    :model-value="toggle"
```